CHANGELOG.md

```diff
@@ -7,6 +7,24 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased](https://github.com/openghg/openghg/compare/0.5.1...HEAD)
 
+### Added
+
+- Added `standardise_column()` wrapper function - [PR #569](https://github.com/openghg/openghg/pull/643)
+- The 'height_name' definition from the [openghg/supplementary_data repository](https://github.com/openghg/supplementary_data) for each site can now be accessed, used and interpreted - [PR #648](https://github.com/openghg/openghg/pull/648)
+- Multiple object stores are now supported. Any number of stores may be accessed and read from and written to - [PR #664](https://github.com/openghg/openghg/pull/664)
+- Allow metadata within metastore to be updated for existing data sources for the latest version, start and end dates of the data - [PR #652](https://github.com/openghg/openghg/pull/652)
+- Allow mismatches between values in metadata and data attributes to be updated to either match to the metadata or attribute collated details - [PR #682](https://github.com/openghg/openghg/pull/682)
+
+### Changed
+
+- Configuration file format has been updated to support multiple object stores, note directing users to upgrade has been added
+- The `openghg --quickstart` functionality has been updated to allow multiple objects stores to be added and migrate users from the previous version of the configuration file
+
+### Removed
```

Comment:
Did you want to add something in ###Changed about your moved from `DataHandler` to `DataManager` or do you feel this would be too much in the details?

Reply:
Done